apps/platform/src/components/integrations/integrationList/index.tsx

```diff
@@ -72,9 +72,11 @@ function IntegrationList() {
   const handleClick = useCallback(
     (integration: Integration) => {
       setSelectedIntegration(integration)
-      router.push(`/integrations?tab=details`)
+      if (selectedWorkspace?.slug && integration.slug) {
+        router.push(`/${selectedWorkspace.slug}/${integration.slug}`)
```

Comment:
I think if we make the route `/${selectedWorkspace.slug}/integrations/${integration.slug}` then ig it will look better